app/views/layouts/submissions/editor/_playground_form.html.erb

```diff
@@ -1,8 +1,8 @@
 <div class="console">
 </div>
-<div class="pull-left">
-  <div class="btn btn-info clear-console"> <%= t :clear_console %></div>
+<div class="console-overlapped">
+  <a class="clear-console"><%= fa_icon('trash-o', text: t(:reset))%></a>
```

Comment:
Also please notice that this line

```
fa_icon('trash-o', text: t(:reset))
```

is duplicated. 

We could add a helper `reset_button`